xwalk_android_tests.gypi

```diff
@@ -248,12 +259,18 @@
       'dependencies': [
         'xwalk_app_runtime_java',
         'xwalk_runtime_client_test_utils_java',
+        'libxwalkdummy',
```

Comment:
I do not think this change is related. If you're trying to fix some bug, please submit separate PR